builder/bundles.go

```diff
@@ -1181,3 +1187,65 @@ func getClosestAncestorOwner(path string) (int, int, error) {
 	gid := int(fi.Sys().(*syscall.Stat_t).Gid)
 	return uid, gid, nil
 }
+
+func writeOsReleaseFile(filename string, chrootDir string) error {
+	var f *os.File
+	err := os.MkdirAll(filepath.Dir(filename), 0655)
+	if err != nil {
+		return err
+	}
+	f, err = os.Create(filename)
+	if err != nil {
+		return err
+	}
+
+	err = f.Chmod(0644)
+	if err != nil {
+		return err
+	}
+
+	_, err = fmt.Fprintln(f, "NAME="+"\""+"Clear Linux OS"+"\"")
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintln(f, "ID="+"clear-linux-os")
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintln(f, "ID_LIKE="+"clear-linux-os")
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintln(f, "VERSION_ID="+"")
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintln(f, "PRETTY_NAME="+"\""+"Clear Linux OS"+"\"")
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintln(f, "ANSI_COLOR="+"\""+"1;35"+"\"")
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintln(f, "HOME_URL="+"\""+chrootDir+"\"")
```

Comment:
I think @bryteise mentioned the default value for "HOME_URL" to be the `Swupd.CONTENTURL` in the builder.conf